library/core/src/intrinsics/mir.rs

```diff
@@ -110,15 +110,15 @@
 //!         let popped;
 //!
 //!         {
-//!             Call(_unused = Vec::push(v, value), pop)
+//!             Call(_unused = Vec::push(v, value), pop, Continue())
 //!         }
 //!
 //!         pop = {
-//!             Call(popped = Vec::pop(v), drop)
+//!             Call(popped = Vec::pop(v), drop, Continue())
```

Comment:
This makes me wonder if we should replace `Call(popped = Vec::pop(v), drop)` by `Call(popped = Vec::pop(v), ReturnTo(drop))` or something like that... basically emulating named arguments. That's orthogonal to your PR though so I'd also be happy to do it as a follow-up if you prefer.